### ISSUE TITLE
I can not run the program

### ISSUE THREAD

**Issue by hvashchenko** [MShaker/python-graph-gui#1]
When I want to run the program, I get an error: 
QObject::connect: Cannot connect UpdateData::signal() to (null)::update_data()
Traceback (most recent call last):
  File "/home/tantalus/dev/envs/graph/GraphGui.py", line 351, in <module>
    ui.setupUi(MainWindow)
  File "/home/tantalus/dev/envs/graph/GraphGui.py", line 250, in setupUi
    self.scene.data_updater.signal.connect(self.update_data)
TypeError: connect() failed between UpdateData.signal[] and update_data()

I use: 
python 3.5
Qt version:  5.8.0
SIP version:  4.19.2
PyQt version:  5.8.2
Please help me with the launch of the program.
Sorry for my English, I'm just from Ukraine

**Comment by hvashchenko** [comment]
QObject::connect: Cannot connect UpdateData::signal() to (null)::update_data()
Traceback (most recent call last):
  File "GraphGuiApplication.py", line 404, in <module>
    ui.setupUi(MainWindow)
  File "GraphGuiApplication.py", line 249, in setupUi
    self.button_setup()
  File "GraphGuiApplication.py", line 264, in button_setup
    self.scene.data_updater.signal.connect(self.update_data)
TypeError: connect() failed between UpdateData.signal[] and update_data()

**Comment by MShaker** [comment]
Hello,

So I'm pretty new to using PyQt and python in general.  This is my first project using PyQt5 so I don't know all the ins and outs.  

I'm using the PyQt that came with my Anaconda3 download and it is version 5.6.0.

The program runs fine on my computer so I am not sure exactly what the problem is, but after looking at your error I played around with the code and found that I could recreate it at line 264 of GraphGuiApplication.py.

My original code was:      **self.scene.data_updater.signal.connect(self.update_data)**
and this worked

when I changed it to:       **self.scene.data_updater.signal.connect(self.update_data())**
I got the same error as you did

So maybe try:     **self.scene.data_updater.signal.connect(lambda: self.update_data())**
Because this also worked for me.

Hopefully that helps!
